test/sdk/ThreadTest.ooc

```diff
@@ -0,0 +1,77 @@
+use ooc-unit
+use ooc-base
+import threading/Thread
+import threading/native/ConditionUnix
+import os/Time
+
+IntObject: class {
```

Comment:
Is that it? We've had lots of problems with Closures, but I never realized it had to do with covers... Might explain a few things!